tests/Runner/Output/JUnitPrinter.phpt

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/**
+ * @phpversion 5.4  Requires constant PHP_BINARY available since PHP 5.4.0
+ */
+
+use Tester\Assert,
+	Tester\Environment,
+	Tester\Runner\Output\JUnitPrinter;
+
+require __DIR__ . '/../../bootstrap.php';
+require __DIR__ . '/../../../Tester/Runner/TestHandler.php';
+require __DIR__ . '/../../../Tester/Runner/Runner.php';
+require __DIR__ . '/../../../Tester/Runner/OutputHandler.php';
+require __DIR__ . '/../../../Tester/Runner/Output/JUnitPrinter.php';
+
+
+Environment::$useColors = FALSE;
+$runner = new Tester\Runner\Runner(createInterpreter());
+$printer = new JUnitPrinter($runner);
+$runner->paths[] = __DIR__ . '/cases/*.phptx';
+$runner->outputHandlers[] = $printer;
+ob_start();
+$runner->run();
+$output = ob_get_clean();
+
+$expected = <<<XML
+<?xml version="1.0" encoding="UTF-8"?>
+<testsuites>
+	<testsuite errors="1" skipped="1" tests="3" time="%a%" timestamp="%a%">
+		<testcase classname="Output/cases/fail.phptx" name="Output/cases/fail.phptx">
```

Comment:
@vysinsky The paths will fail on Windows. You can use [`%ds%`](https://github.com/nette/tester/blob/f999bdac92db8a35d07905ebdd78d383ad7a22f0/Tester/Framework/Assert.php#L35) not documented "yet".